frontend/src/components/commons/Tab.jsx

```diff
@@ -13,6 +13,17 @@ const styles = {
     border: "1px solid transparent",
     borderRadius: "4px 4px 0 0"
   },
+  disabledButton: {
```

Comment:
This should only include additional or overriding styles to `button`.

Then below, you can say `style={{ ...styles.button, ...styles.disabledButton }}`

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/241/commits/653c0bb985ed353d8d17d06c8991ec65f445f9c3